docker/common-services.yaml

```diff
@@ -33,6 +33,7 @@ services:
     image: voxel51/fiftyone-teams-api:v1.7.1
     environment:
       CAS_BASE_URL: ${CAS_BASE_URL:-http://teams-cas:3000/cas/api}
+      FEATURE_FLAG_ENABLE_INVITATIONS: ${FEATURE_FLAG_ENABLE_INVITATIONS:-true}
```

Comment:
This was already 'true' [by default](https://github.com/voxel51/voxel-hub/blob/main/api/fiftyone_teams_api/env.py#L123)

So a customer who did an override in `teams-app` would get invitations enabled all over the place.

Just a note, adding it here makes it easy!